### PR TITLE
Widen issue search when focused

### PR DESCRIPTION
**Pull request by bfred-it**
I feel that the field is too small, especially after adding our `sort:updated-desc`. This should make it more usable

### CODE REVIEW

**Comment by busches** [comment]
Do we want to put these rules behind a custom css flag so we can toggle them off via JS?

**Comment by yakov116** [comment]
@busches it's not like we are hiding anything. Why would a user want to take it off? I think it's very useful since we are filling up almost there entire box with the default sort.

**Comment by bfred-it** [comment]
Most of all, we don't want to put every CSS feature behind a JS flag, especially if they can be overridden with 2 lines of CSS in the options

**Comment by busches** [comment]
I agree with both of you, I just wanted to hash out the direction we plan on going. :)

**Comment by sindresorhus** [comment]
We only intentionally enable disabling a feature for controversial features.

File: source/content.css [hunk 1]
```diff
@@ -509,9 +509,30 @@ div.inline-comment-form .form-actions,
...
-.subnav-search-input[aria-label='Search all issues'] {
-	width: 420px;
+/* Extend issue search input width when focused */
+.subnav-search {
+	width: 320px; /* Original width on GitHub */
+}
+.subnav-search .subnav-search-input {
+	position: relative;
+	z-index: 1;
+	transition: width 0.3s ease-out;
```
Comment (sindresorhus) [inline source/content.css]: Can we make this `0.2s`? 0.3 feels a tiny bit too slow.

**Comment by sindresorhus** [comment]
You hide the icon in https://github.com/pulls?q=is%3Apr+is%3Aopen+author%3Asindresorhus+archived%3Afalse+sort%3Aupdated-desc, but I think you should adjust the left padding too. Right now it's too much and looks a bit strange.

**Comment by sindresorhus** [comment]
If you focus the search bar when at https://github.com/pulls?q=is%3Apr+is%3Aopen+archived%3Afalse+sort%3Aupdated-desc+mentions%3Asindresorhus, you get this:

screen shot 2018-11-18 at 19 03 06

**Comment by yakov116** [comment]
@sindresorhus 	could we merge this?

**Comment by sindresorhus** [comment]
Looks great :)

**Pull request merged**